### PR TITLE
wet-195 Ministerial-profile GCWeb template updates

### PR DESCRIPTION
**Pull request by Ricokola**
Minor updates to the Ministerial profile page template:
- Removal of the following sections: Latest, Recent activities, Twitter feed, Follow us and Gallery
- Creation of a Features optional section v6.1
- Change for 9 col instead of 6
- Creation of a documentation page (Eng and Fra)

### CODE REVIEW

File: templates/ministerial/ministerial-doc-en.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	<dd>Canada.ca template</dd>
+
+	<dt>Last review</dt>
+	<dd>2021-09-15</dd>
+
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+
+	<dt>Specification</dt>
+	<dd><a href="https://design.canada.ca/mandatory-templates/ministerial-profile-pages.html">Ministerial profile pages - Canada.ca mandatory template</a></dd>
```
Comment (GormFrank) [inline templates/ministerial/ministerial-doc-en.html:25]: At the moment, the right link to specifications page ends with a "-draft", like so:
```suggestion
	<dd><a href="https://design.canada.ca/mandatory-templates/ministerial-profile-pages-draft.html">Ministerial profile pages - Canada.ca mandatory template</a></dd>
```

File: templates/ministerial/ministerial-doc-en.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	<li><a href="ministerial-fr.html" lang="fr">Exemple pratique</a></li>
+</ul>
+
+<h2>Latest changes</h2>
+<ul>
+	<li>replaced Latest, Recent announcements, and Gallery with a flexible Features section </li>
+	<li>removed the social media feed and added an optional Follow me section instead</li>
+	<li>simplified and streamlined the guidance </li>
+	<li>clarified how to display ministers, parliamentary secretaries and heads of organizations</li>
+</ul>
```
Comment (GormFrank) [inline templates/ministerial/ministerial-doc-en.html:43]: Add an H3 before the list mentioning: "Version 2.0", and then an H3 after the list mentioning: "Version 1.0" followed by a single bullet list mentioning: "Initial template", something like so:
```suggestion
<h3>Version 2.0</h3>
<ul>
	<li>replaced Latest, Recent announcements, and Gallery with a flexible Features section </li>
	<li>removed the social media feed and added an optional Follow me section instead</li>
	<li>simplified and streamlined the guidance </li>
	<li>clarified how to display ministers, parliamentary secretaries and heads of organizations</li>
</ul>
<h3>Version 1.0</h3>
<ul>
	<li>Initial template</li>
</ul>
```

File: templates/ministerial/ministerial-en.html [hunk 3]
```diff
@@ -33,113 +28,26 @@ <h2>Contact information</h2>
 				<strong>Email:</strong> <a href="mailto:">[first.last@canada.ca]</a> <span class="glyphicon glyphicon-envelope"></span></p>
 		</section>
 	</div>
-	<div class="col-md-3">
-		<h2>Twitter</h2>
-		[Twitter feed]
-	</div>
 </div>
 <div class="row">
```
Comment (GormFrank) [inline templates/ministerial/ministerial-en.html:32]: Please remove this wrapping div (and don't forget its closing tag), it makes the content overflow on both sides.
```suggestion

```

**Review by GormFrank — CHANGES_REQUESTED** [review]
@Ricokola Few changes and can you please include the "Share this page" button at the bottom of the page?

File: templates/ministerial/ministerial-doc-en.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+	<li>Initial template</li>
+</ul>
+
+<h2>Latest changes</h2>
+<ul>
+	<li>replaced Latest, Recent announcements, and Gallery with a flexible Features section </li>
+	<li>removed the social media feed and added an optional Follow me section instead</li>
+	<li>simplified and streamlined the guidance </li>
+	<li>clarified how to display ministers, parliamentary secretaries and heads of organizations</li>
+</ul>
```
Comment (GormFrank) [inline templates/ministerial/ministerial-doc-en.html:53]: This shoud be the outcome, and maybe it is clearer if you replace "Recent hanges" by "Change history":
```suggestion
<ul>
	<li><a href="ministerial-en.html">Working example</a></li>
	<li><a href="ministerial-fr.html" lang="fr">Exemple pratique</a></li>
</ul>

<h2>Change history</h2>

<h3>Version 2.0</h3>
<ul>
	<li>replaced Latest, Recent announcements, and Gallery with a flexible Features section </li>
	<li>removed the social media feed and added an optional Follow me section instead</li>
	<li>simplified and streamlined the guidance </li>
	<li>clarified how to display ministers, parliamentary secretaries and heads of organizations</li>
</ul>

<h3>Version 1.0</h3>
<ul>
	<li>Initial template</li>
</ul>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
One change remaining, please do the same in French. I'd say that "Change history" should translate to "Historique des changements".

File: templates/ministerial/ministerial-doc-en.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
+---
+title: Documentation about Ministerial profile page template
+description: Documentation and working examples for the Ministerial profile tempalte.
+language: en
+altLangPage: ministerial-fr.html
```
Comment (GormFrank) [inline templates/ministerial/ministerial-doc-en.html:5]:
```suggestion
altLangPage: ministerial-doc-fr.html
```

File: templates/ministerial/ministerial-doc-fr.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
+---
+title: Documentation au sujet de la page de profil des ministres
+description: Documentation et exemple pratique des pages de profil des ministres.
+language: en
```
Comment (GormFrank) [inline templates/ministerial/ministerial-doc-fr.html:4]:
```suggestion
language: fr
```

File: templates/ministerial/ministerial-doc-fr.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+altLangPage: ministerial-fr.html
```
Comment (GormFrank) [inline templates/ministerial/ministerial-doc-fr.html:5]:
```suggestion
altLangPage: ministerial-doc-en.html
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
Last changes to the front matter and please include the following [built _data/templates.json file](https://gormanproductions.ca/wet/labs/pr1884/templates.json) to this PR, then it'll be good to go! :)

**Review by GormFrank — APPROVED** [review]
@duboisp Take a look if you wish, but I believe we are good to go! This has to be merged today :)

File: templates/ministerial/ministerial-en.html [hunk 8]
```diff
@@ -33,113 +29,24 @@ <h2>Contact information</h2>
...
-			</a>
-		</div>
-		<div class="col-lg-3 col-md-6 mrgn-bttm-lg">
-			<a href="#">
-				<figure>
-					<img src="img/250x250.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
-					<figcaption>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum, fugiat!</figcaption>
-				</figure>
-			</a>
+<section class="gc-features">
```
Comment (duboisp) [inline templates/ministerial/ministerial-en.html:33]: Please use the latest pattern for "gc-features", Although I understand we need this page template to work with v9.4.1 of GCWeb.

**Review by duboisp — APPROVED** [review]
@GormFrank I am going to approve considering we need this template to be compatible with v9.4.1 of GCWeb because that is the version currently deployed on Canada.ca. 

We would need to review this template and make it more modular where the component code are taken directly from component working space instead of being duplicated here. @Ricokola I would help you to do that.

The documentation would need to be reviewed and all the component in this template would need to be clearly identify.

**Pull request merged**